### PR TITLE
Patch graceful shutdown bug

### PR DESCRIPTION
**Pull request by zain-kabani**
Fixes graceful shutdown bug for startup and cancel type ClientConnectionTypes

Adds logging for active clients when shutting down

### CODE REVIEW

File: src/client.rs [hunk 1]
```diff
@@ -204,7 +204,7 @@ pub async fn client_entrypoint(
                 Ok(mut client) => {
                     info!("Client {:?} connected (plain)", addr);
 
-                    if client.is_admin() {
+                    if !client.is_admin() {
```
Comment (levkk) [inline src/client.rs:207]: This is the opposite of what we're doing now. Could you describe the bug you're seeing?

Comment (zainkabani) [reply src/client.rs:207]: Recreating the bug.

Start pgcat.

Create client with sslmode disabled, `?sslmode=disable`
execute single query then close connection.

Send sigint to pgcat

Comment (zainkabani) [reply src/client.rs:207]: this is actually most likely a typo since in every other case we're doing `!client.is_admin()`

Comment (levkk) [reply src/client.rs:207]: You're right, very nice find.

**Pull request merged**